src/components/structures/GroupView.js

```diff
@@ -862,6 +879,16 @@ export default React.createClass({
                 <h3> { _t("Long Description (HTML)") } </h3>
                 <textarea
                     value={this.state.profileForm.long_description}
+                    placeholder={_t(
```

Comment:
I wonder if this monster would be easier to read as a constant which can be defined at the top level somewhere?  e.g.:

```
const LONG_DESC_PLACEHOLDER = _t("<h1>HTML for your community's page</h1>
<p>
    Use the long description to introduce new members to the community, or distribute
    some important <a href="foo">links</a>
</p>
<p>
    You can even use 'img' tags
</p>
";
'''

...feels more legible to me?

Reply:
wfm